mkdocs.yml

```
@@ -189,15 +189,15 @@ nav:
           - 'Recovery': variants/protectli_vp46xx/recovery.md
           - 'Hardware Configuration Matrix': variants/protectli_vp46xx/hardware-matrix.md
           - 'Test matrix': variants/protectli_vp46xx/test-matrix.md
-      # - 'Protectli PT201/PT401/PT601':
-      #     - 'Overview': variants/protectli_ptx01/overview.md
-      #     - 'Releases': variants/protectli_ptx01/releases.md
-      #     - 'Building manual': variants/protectli_ptx01/building-manual.md
-      #     - 'Initial deployment': variants/protectli_ptx01/initial-deployment.md
-      #     - 'Firmware update': variants/protectli_ptx01/firmware-update.md
-      #     - 'Recovery': variants/protectli_ptx01/recovery.md
-      #     - 'Hardware Configuration Matrix': variants/protectli_ptx01/hardware-matrix.md
-      #     - 'Test matrix': variants/protectli_ptx01/test-matrix.md
+      - 'Protectli V1210/V1410/V1610':
+          - 'Overview': variants/protectli_v1x10/overview.md
+          - 'Releases': variants/protectli_v1x10/releases.md
+          - 'Building manual': variants/protectli_v1x10/building-manual.md
+          - 'Initial deployment': variants/protectli_v1x10/initial-deployment.md
+          - 'Firmware update': variants/protectli_v1x10/firmware-update.md
+          - 'Recovery': variants/protectli_v1x10/recovery.md
+          - 'Hardware Configuration Matrix': variants/protectli_v1x10/hardware-matrix.md
+          - 'Test matrix': variants/protectli_v1x10/test-matrix.md
```

```suggestion
      # - 'Protectli PT201/PT401/PT601':
      #     - 'Overview': variants/protectli_ptx01/overview.md
      #     - 'Releases': variants/protectli_ptx01/releases.md
      #     - 'Building manual': variants/protectli_ptx01/building-manual.md
      #     - 'Initial deployment': variants/protectli_ptx01/initial-deployment.md
      #     - 'Firmware update': variants/protectli_ptx01/firmware-update.md
      #     - 'Recovery': variants/protectli_ptx01/recovery.md
      #     - 'Hardware Configuration Matrix': variants/protectli_ptx01/hardware-matrix.md
      #     - 'Test matrix': variants/protectli_ptx01/test-matrix.md
```

This must still stay hidden